src/System.Collections.NonGeneric/tests/System.Collections.NonGeneric.Tests.csproj

```diff
@@ -147,14 +147,15 @@
     <Compile Include="$(CommonTestPath)\System\Diagnostics\DebuggerAttributes.cs">
       <Link>Common\System\Diagnostics\DebuggerAttributes.cs</Link>
     </Compile>
-  </ItemGroup>
-  <!-- Performance Tests -->
-  <ItemGroup Condition="'$(RunPerfTestsForProject)' == 'true'">
+    <!-- Performance Tests -->
     <Compile Include="Performance\Perf.HashTable.cs" />
     <Compile Include="$(CommonTestPath)\System\PerfUtils.cs">
       <Link>Common\System\PerfUtils.cs</Link>
     </Compile>
   </ItemGroup>
+  <PropertyGroup Condition="'$(Performance)' == 'true'">
```

Comment:
@ericeil wrote the benchmark so he'll know better than me, but I think that including it would disable concurrent running of the tests and therefore be an unnecessary perf hit for execution time of the unit tests. I'm not sure if this occurs in [Fact]s and [Theory]s.

Now that I've had some time to think about this I'm also curious about how Benchmark iterations are chosen when running a Benchmark through Corerun instead of through the xunit.performance runner. Do we still do ~1000 iterations in the Benchmark.Iterations loop even if they're not being recorded?